test/functional/apps/discover/_discover.ts

```diff
@@ -20,14 +20,17 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const inspector = getService('inspector');
   const elasticChart = getService('elasticChart');
   const PageObjects = getPageObjects(['common', 'discover', 'header', 'timePicker']);
+
   const defaultSettings = {
     defaultIndex: 'logstash-*',
   };
 
   describe('discover test', function describeIndexTests() {
     before(async function () {
       log.debug('load kibana index with default index pattern');
-      await esArchiver.load('discover');
+
+      await kibanaServer.savedObjects.clean({ types: ['search'] });
```

Comment:
@spalger what's the deal with this guy?  is it like `await esArchiver.load('empty_kibana')`?

Reply:
Essentially, but it's more targeted, doesn't require running a migration, and only cleans out objects of a specific type

Reply:
Ah, very nice!

Reply:
@spalger Hey chief, there will be times when we need to run migrations correct?

Reply:
Nope, the esArchiver API automatically migrates documents as they're imported.